pca.go

```diff
@@ -0,0 +1,79 @@
+// Copyright ©2016 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package stat
+
+import (
+	"math"
+
+	"github.com/gonum/floats"
+	"github.com/gonum/matrix"
+	"github.com/gonum/matrix/mat64"
+)
+
+// PrincipalComponents returns the principal component direction vectors of
```

Comment:
You could say "... returns the principle component direction vectors and the variances of the principle component scores, vecs * a, computed using the SVD of the input a" as vars are infact the variances of the columns of vecs * a (where * is matrix multiplication, and assuming a is centred already).